lib/advisor/core/answers/answers.ex

```diff
@@ -0,0 +1,60 @@
+defmodule Advisor.Core.Answers do
+  alias Advisor.Repo
+  alias Advisor.Core.Answer
+  import Ecto.Query
+
+  @into_list []
+
+  def store(params) do
+    Repo.insert_all(Answer, all_answers_in(params), returning: true)
+  end
+
+  def all_answers_in(%{"id" => advice_request_id} = params) do
+    params
+    |> Enum.reduce(@into_list, &to_answer/2)
+    |> add(advice_request_id)
+  end
+
+  defp add(answers, advice_request_id) when is_list(answers) do
+    answers
+    |> Enum.map(&(Map.put(&1, :advice_request_id, advice_request_id)))
+  end
+
+  defp to_answer({question_id, answer}, answers) do
```

Comment:
I would move this method before `defp add(` as it is called before in `all_answers_in`